### ISSUE TITLE
[FR] Graph export

### ISSUE THREAD

**Issue by dleeftink** [SkepticMystic/graph-analysis#42]
Hi Skeptic, thanks for the great plugin. Usability wise, it would be really helpful to able to export the co-citations table as a plain node/edge list or adjacency matrix (either for the local note or the global graph). This way, external graphing becomes possible im GraphViz, yEd, d3.force etc. using simple copy-paste.

(Or maybe even cooler, extracting hierarchies from flat notes by applying breadth first search or other spanning tree algorithms to the edge list)

**Comment by SkepticMystic** [comment]
Interesting idea, we can definitely look into this. I think we should definitely use a different lib (as per #44), so we might be other issues on hold until we've refactored everything

**Comment by dleeftink** [comment]
Nice one, I think the functionality is already somewhat in place if an "unfold all" button was added to the co-citations table. The results could than simply be copied elsewhere.

**Comment by dleeftink** [comment]
On that note, the co-citation table is also a bit finicky on mobile, as the row unfold button rests closely to the note links. Maybe an option to deactive links in the table view would suffice to prevent accidental presses.

**Comment by HEmile** [comment]
> On that note, the co-citation table is also a bit finicky on mobile, as the row unfold button rests closely to the note links. Maybe an option to deactive links in the table view would suffice to prevent accidental presses.

I agree with this, btw. An easy workaround is to click on the numbers that give the co-citation strength, that will also expand the thing. But yeah, should probably be easier...
